.github/workflows/ci.yml

```diff
@@ -79,14 +79,37 @@ jobs:
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $MAVEN clean install -B --strict-checksums -V -T 1C -DskipTests -P ci -pl '!:trino-server-rpm'
```

Comment:
it doesn;t have to be `install` anymore (AFAICT was needed for jdbc tests)

i think it can be `verify`